src/components/text-field.js

```diff
@@ -3,48 +3,74 @@ import PropTypes from 'prop-types'
 import clsx from 'clsx'
 
 import { InputBase } from '../base-components'
+import Area from './area'
 
 
-const _textFieldClasses = ({ error }) => ({
-  label: clsx('font-sans text-sm ml-1', { 'text-primary-700': !error, 'text-error': error }),
-  helperText: clsx('font-sans text-xs ml-1', { 'text-primary-700': !error, 'text-error': error }),
+const _textFieldClasses = ({ container, success, error }) => ({
+  container: `${container ? container : 'w-250px flex flex-col'}`,
+  label: 'text-secondary-600',
+  helperText: clsx('mt-1.5 text-secondary-600', { 
+    'text-error-500': error, 
+    'text-success-500': success, 
+  }),
+  wordCount: 'mt-1.5 col-start-2 justify-self-end text-secondary-600 text-xxs tracking-lg leading-1.6',
 })
-const _inputBaseClasses = ({ focus, error, width, filled }) => ({
-  root: clsx(`${width} rounded-sm p-sm`,
-    { 'border-primary-700 shadow-focused-primary': focus && !error },
-    { 'border-error shadow-focused-error': focus && error },
-    { 'border-primary-700 bg-neutral-50': filled },
+
+const _inputBaseClasses = ({ focus, success, error, root, filled, disabled }) => ({
+  root: clsx(`${root ? root : 'h-9 mt-1.5 rounded-sm p-sm font-sans text-sm tracking-sm leading-1.43'}`,
+    { 'border-secondary-600': !disabled },
+    { 'border-interactive-500 shadow-focused-interactive': focus && !error && !success },
+    { 'border-error-500 shadow-focused-error': error },
```

Comment:
The shadow seems not right for`interactive`, `error` and `success`.
Compared with the design, it's a little bit too much for the focus ring. Check the difference.

Designed:
![image](https://user-images.githubusercontent.com/20359409/126835978-91c0f952-ae7b-43b5-8349-4ced5ed53540.png)
In PR
![image](https://user-images.githubusercontent.com/20359409/126835934-961006a0-ea21-4161-a1af-e2a1aadb25e2.png)